core/src/main/java/org/elasticsearch/rest/action/cat/RestRecoveryAction.java

```diff
@@ -90,7 +90,8 @@ protected Table getTableWithHeader(RestRequest request) {
                 .addCell("source_host", "alias:shost;desc:source host")
                 .addCell("target_host", "alias:thost;desc:target host")
                 .addCell("repository", "alias:rep;desc:repository")
-                .addCell("snapshot", "alias:snap;desc:snapshot")
+                .addCell("snapshot_name", "alias:snapName;desc:snapshot_name")
+                .addCell("snapshot_uuid", "alias:snapUUID;desc:snapshot_uuid")
```

Comment:
we only have index name (and not index uuid) in this one, wonder if we need snapshot uuid here...